fs/dafsclient/rmtfile.cpp

```diff
@@ -2557,31 +2557,41 @@ class CStorageApiInfo : public CInterfaceOf<IStorageApiInfo>
     {
         return xml->queryProp("@type");
     }
-    virtual const char * queryStorageApiAccount() const override
+    virtual const char * queryStorageApiAccount(unsigned stripeNumber) const override
     {
-        return xml->queryProp("@account");
+        const char *account = xml->queryProp("@account");
```

Comment:
I think this is more natural the other way around.  Check the container first, then the main information.  It will only make a difference if the account is specified in both places - but if it is you would expect the entry on the container entry to take precedence, and the entry here to be the default.